.drone.yml

```diff
@@ -16,7 +16,7 @@ steps:
   - git fetch --depth 100 origin $DRONE_TARGET_BRANCH
 
 - name: verify-pr
-  image: joshlory/code-dot-org:0.10
+  image: wjordan/code-dot-org:2020-09-10
```

Comment:
Could be a good chance to set up a shared dockerhub account for our team and put this there. NBD if you don't want to do that now though.